packages/web/src/components/gcds-error-summary/gcds-error-summary.tsx

```diff
@@ -119,11 +123,11 @@ export class GcdsErrorSummary {
    */
   sortErrors() {
     const sortable = [];
-    for (const id in this.errorLinksObject) {
+    for (const [key, value] of Object.entries(this.errorLinksObject)) {
       sortable.push([
-        id,
-        this.errorLinksObject[id],
-        document.querySelector(id).getBoundingClientRect().y,
+        key,
+        value,
+        (value as HTMLElement).getBoundingClientRect().y,
```

Comment:
Could you use `map` instead of `forEach` to create a new object that is a little bit more concise?
Instead of manually creating an object with `forEach`, you can use `Object.fromEntries` to create the object from an array of key-value pairs.

Picturing something like this (full disclosure - didn't test this):

```
sortErrors() {
  const sortable = Object.entries(this.errorLinksObject).map(([key, value]) => [
    key,
    value,
    (value as HTMLElement).getBoundingClientRect().y,
  ]);

  sortable.sort((a, b) => a[2] - b[2]);

  return Object.fromEntries(sortable);
}
```